### ISSUE TITLE
Auth with `password` bypass 2nd factor authentication

### ISSUE THREAD

**Issue by duyleekun** [bizflycloud/gobizfly#91]
It seems auth using `password` skips OTP check.

**Comment by greatbn** [comment]
Yes. This issue we did considered recently. We will design the flow to enable 2nd factor auth when using the API to get token.

**Comment by duyleekun** [comment]
And with the issue in #90 , I couldn't find a secured way to store/share Bizfly credential. Is there any alternative?
